src/apis/platformvm/interfaces.ts

```diff
@@ -66,6 +66,13 @@ export interface GetCurrentValidatorsParams {
   nodeIDs?: string[]
 }
 
+export interface GetClaimablesParams {
+  addresses: string[]
+  depositTxIDs: string[]
+  locktime: BN
```

Comment:
If this interface is request params, we don't use BN anymore -> pls. convert use string